_rules/document-title-is-descriptive.md

```diff
@@ -0,0 +1,200 @@
+---
+name: Document title is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the document title describes the topic or purpose of the document.
+
+success_criterion: 
+- 2.4.2 # Page Titled (A)
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Corbb O’Connor 
+
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to the [document title](https://www.w3.org/TR/html5/single-page.html#dom-tree-accessors) in a [document](https://www.w3.org/TR/dom/#concept-document) where the [document element](https://www.w3.org/TR/dom/#document-element) is an HTML `<html>` element **and** the `<title>` element is [non-empty](#non-empty).
+
+**Note:** Due to the definition of [document title](https://www.w3.org/TR/html5/single-page.html#dom-tree-accessors) this rule only applies to the first `<title>` element in the document, even if there are more `<title>` elements in the document.
+
+### Expectation
+
+The target element describes the topic or purpose of the overall content of the [document](https://www.w3.org/TR/dom/#concept-document).
+
+## Assumptions
+
+_There are no assumptions for this rule._
+
+## Accessibility Support
+
+- This rule assumes that browsers only recognize the first `<title>` element if multiple `<title>` elements are present in the [document](https://www.w3.org/TR/dom/#concept-document). Testing shows that this in general is the case. Therefore the scope of this rule is limited to only checking the first `<title>` element in a document.
+
+## Background
+
+- [Understanding Success Criterion 2.4.2: Page titled](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html)
+- This rule is testing [Technique G88: Providing descriptive titles for Web pages](https://www.w3.org/WAI/WCAG21/Techniques/general/G88)
+- A prerequisite for this rule is that [Technique H25: Providing a title using the title element](https://www.w3.org/WAI/WCAG21/Techniques/html/H25) is used for living up to 2.4.2 Page Titled
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+The ´<title>´ element describes the content of the document.
```

Comment:
Edit the ´ (apostrophe) to ` (backquote) to have the title element displayed as code.